### ISSUE TITLE
What is the best config to achieve most profit?

### ISSUE THREAD

**Issue by rinkal26783** [RainbowMiner/RainbowMiner#143]
I have tried all possible settings on different pools. In rainbowminer it shows profit of 220+ XVG but in actual i get only around 100 xvg. This is so below to profit line.
Please help.

**Comment by RainbowMiner** [comment]
It depends all heavily on the luck at a pool and the current popularity(=hashrate) of a coin (in the current bear market, this factor suffers from huge swings). So, the expected return leads to inaccurate results, if either the luck or the hashrate has huge swings.

Sometimes multiple blocks are found in a row, which shoots up the current estimated prices, and then, sometimes not a single block pops up thus leaving you mining for nothing. Having this in mind, I am using three setup parameters (ok, it is all a bit of alchemy ;-) )
- first, I set the DataWindow for all pools in pools.config.txt to "average-2e". That one uses an average of the estimate values that the pool's report (estimate_current + estimate_24h)/2. I have made the experience, that this does a good job on averaging out spices, peeks and pokes and nears the estimated profit a bit more to reality.
- second, I select pools with constant high hashrates for my selected algorithm, only. I avoid mining, when the hashrate sometimes comes done to zero. This reduces the luck-no-luck factor by a huge degree and causes a constant income stream. Basicly spoken, stick with the mass.
- third, my pool set always contains NiceHash, because from time to time there pop up very good paying algorithm. And this pool pays for your hashrate, so no need to wait for a block being found.
